README.md

```diff
@@ -70,9 +70,45 @@ Main features of diktat are the following:
 To **autofix** all code style violations use `-F` option.
 
 ## Run with Maven
+### Use the diktat-maven-plugin
+This plugin is available since version 0.1.3. You can see how it is configured in our project for self-checks: [pom.xml](pom.xml).
```

Comment:
better to give link to our examples-repo and put the example there

Reply:
We have #264, maybe it's time to move examples to main repo and run functional tests on them. It could be done after gradle plugin is released.